docs/conf.py

```diff
@@ -26,8 +27,20 @@
 # -- Options for HTML output -------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
-html_theme = 'alabaster'
-html_static_path = []
+html_theme = 'sphinx_book_theme'
+html_title = "JAX AI Stack"
```

Comment:
Please change to single quotes for consistency (and to satisfy the lint checker)

Reply:
Thanks, done!